docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
+   ```
+
+</TabItem>
+<TabItem value="2" label="Terraform">
+Access your new service with the Apache Cassandra client using the environment variables
+assigned to Terraform outputs:
 
 ```bash
-avn service create demo-cassandra       \
-   --service-type cassandra             \
-   --cloud CLOUD_AND_REGION             \
-   --plan PLAN                          \
-   --project PROJECT_NAME
+./cqlsh --ssl --host=$CASSANDRA_HOST --port=$CASSANDRA_PORT --user=$CASSANDRA_USER --password=$CASSANDRA_PASSWORD
 ```
 
-:::note
+</TabItem>
+<TabItem value="3" label="cqlsh client">
+[Connect to your new service with CLI](/docs/products/cassandra/howto/connect-cqlsh-cli)
+using the
+[the `cqlsh` client](https://cassandra.apache.org/doc/stable/cassandra/tools/cqlsh.html).
+</TabItem>
+</Tabs>
 
-- To preview the full list of default flags, run `avn service create -h`.
-- To list the type-specific options, run `avn service types -v`.
+:::tip
+Discover more tools for connecting to Aiven for Apache Cassandra in
+[Connect to Aiven for Apache Cassandra®](/docs/products/clickhouse/howto/list-connect-to-service).
+:::
+
+## Create keyspaces and tables
+
+Use `cqlsh` to create a keyspace and a table where you can insert your data next.
+
+1. Create a keyspace, for example `library`:
+
+   ```sql
+   CREATE KEYSPACE IF NOT EXISTS library WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : '3' };
+   ```
+
+1. Create a table within the `library` keyspace, for example `book_tracker`:
+
+   ```sql
+   CREATE TABLE IF NOT EXISTS library.book_tracker (book_id text PRIMARY KEY, item_count int, genre text, status text, last_update_timestamp timestamp);
+   ```
+
+## Insert data
+
+Use `cqlsh` to write data in the `book_tracker` table, for example:
+
+```sql
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('01234', 6, 'true_crime', 'available', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('56789', 13, 'romance', 'unavailable', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('12345', 55, 'spirituality_religions', 'available', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('67890', 3, 'adventure', 'unavailable', toTimeStamp(now()));
+```
+
+:::tip
+For importing large amount of data or for batch data load, see:
+
+- [Use DSBULK to load, unload, and count data](/docs/products/cassandra/howto/use-dsbulk-with-cassandra)
+- [Migrate to Aiven with the ZDM Proxy](/docs/products/cassandra/howto/zdm-proxy)
+- [Migrate to Aiven with `sstableloader`](https://aiven.io/blog/aiven-now-supports-sstableloader)
 
 :::
+
+## Read data
+
+Use `cqlsh` to read data from the `book_tracker` table, for example:
+
+- ```sql
+  SELECT * FROM library.book_tracker;
```

Comment:
could you add a `WHERE` clause to the queries? the "SELECT everything" is a bad antipattern in Cassandra world, it requires the "ALLOW FILTERING" to suppress the built-in Cassandra safety mechanisms that fail these kinds of queries; a good query would look like `SELECT ... WHERE book_id = 01234`

Reply:
Thanks @dmitry-potepalov ! There are already two `WHERE` examples that follows this one. Also, I tested this example and it worked fine for me as is. Please let me know if you still think this example is not ok and I'll remove it (adding another `WHERE` example would only add redundancy I guess).

Reply:
Let's leave the first "select all" example and change the `WHERE` clause to reference the primary key of the table (book_id). The queries that require `ALLOW FILTERING` are an antipattern we don't want to promote, they'll get progressively slower and slower as the cluster grows since they don't allow the database to select the right node for the query

Reply:
I updated the first example but I'm not sure how to modify the other ones. Could you please comment inline on each of them and provide your suggestions for improvement?